apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -904,26 +850,30 @@ impl Telemetry {
             }
             Err(err) => {
                 metric_attrs.push(KeyValue::new("status", "500"));
-
-                ::tracing::info!(
-                    monotonic_counter.apollo.router.operations = 1u64,
-                    http.response.status_code = 500i64,
+                u64_counter!(
+                    "apollo.router.operations",
+                    "The number of graphql operations performed by the Router",
+                    1,
+                    "http.response.status_code" = 500
                 );
                 Err(err)
             }
         };
 
         // http_requests_total - the total number of HTTP requests received
-        metrics
-            .http_requests_total
-            .add(&opentelemetry::Context::current(), 1, &metric_attrs);
+        u64_counter!(
+            "apollo_router_http_requests_total",
+            "Total number of HTTP requests made.",
+            1,
+            metric_attrs
+        );
 
-        metrics.http_requests_duration.record(
-            &opentelemetry::Context::current(),
+        f64_histogram!(
```

Comment:
Is that a rename? (appending `_seconds`)

Reply:
It's not a rename. The original code is referencing `metrics.http_requests_duration` which is declared as: `apollo_router_http_request_duration_seconds`.